features/step_definitions/background_steps.rb

```diff
@@ -0,0 +1,5 @@
+Given("the following article exists") do |table|
+    table.hashes.each do |article_hash|
+        FactoryBot.create(:article, article_hash)
```

Comment:
You can shorten this to omit `FactoryBot`  You are including the Syntax methods in you Cucumber configuration ( `features/support/env.rb/)

Reply:
fixed